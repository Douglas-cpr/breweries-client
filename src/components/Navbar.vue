<script setup lang="ts"></script>

<template>
  <nav class="navbar">
    <h1 class="navbar__logo">Breweries</h1>
    <p class="navbar__brand">A breweries list by Open Brewery DB</p>
  </nav>
</template>

<style scoped>
.navbar {
  height: auto;
  width: 100%;
  background-color: var(--navbar-bg);
  color: var(--navbar-color);
  padding: 4% var(--default-width-padding);
}

.navbar__brand {
  color: var(--navbar-subtitle);
}
</style>
