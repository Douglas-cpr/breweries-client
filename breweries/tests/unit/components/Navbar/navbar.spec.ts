import Navbar from '@/components/Navbar.vue'
import { shallowMount } from '@vue/test-utils'

describe('<Navbar />', () => {
  it('should render correctly', () => {
    const wrapper = shallowMount(Navbar)
    
    expect(wrapper.html()).toMatchSnapshot()
  })
})