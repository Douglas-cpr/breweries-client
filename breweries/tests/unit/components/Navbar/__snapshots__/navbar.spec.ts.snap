// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> should render correctly 1`] = `
"<nav class=\\"navbar\\">
  <h1 class=\\"navbar__logo\\">Breweries</h1>
  <p class=\\"navbar__brand\\">A breweries list by Open Brewery DB</p>
</nav>"
`;
